src/os/bluestore/BlueFS.cc

```diff
@@ -712,10 +712,25 @@ void BlueFS::_init_alloc()
 {
   dout(20) << __func__ << dendl;
 
+  auto change_alloc_size = [this](uint64_t& max_alloc_size,
+                                  uint64_t new_alloc, bool& changed) {
```

Comment:
I think it's better to emphasize that 'changed' should keep its previous value if no actual modification occurred.

Reply:
@ifed01 Thank you for review. Do you mean adding a comment?

Reply:
yeah, better to have a comment